.github/workflows/k8s_deploy.yml

```
@@ -57,6 +57,14 @@ on:
                 type: string
                 required: false
                 description: 'image pull secret to use (default: ub-acr-secret)'
+            image_tag_to_deploy:
+                type: string
+                required: false
+                description: Tag of the image, take ref_name if not set. Useful to deploy pushserver from backend repo.
+            config_map_refix:
```

```suggestion
            config_map_prefix:
```
Probably prefix right?